asciidoctor.css

```diff
@@ -1,409 +0,0 @@
-/* Asciidoctor default stylesheet | MIT License | http://asciidoctor.org */
```

Comment:
I'm guessing that this file is not used anymore (it was somehow useful when we were using `convert` instead of `convertFile`, right ?

Reply:
Correct. This is now satisfied by asciidoctor.js.